src/main.rs

```diff
@@ -459,8 +449,34 @@ extern "C" fn kernel_init(hart_id: u64, fdt_addr: u64) {
     let guest_phys_size = mem_map.regions().last().unwrap().end().bits()
         - mem_map.regions().next().unwrap().base().bits();
 
+    // Parse the user-mode ELF containing the user-mode task.
+    let user_elf = include_bytes!("../target/riscv64gc-unknown-none-elf/release/umode");
```

Comment:
Ugh, this is gross but I don't immediately have a better suggestion.

Reply:
Yes, it is indefensible but also the best thing we can do now. Double-checked with @dgreid about this as well.

Reply:
yes, it's just a hack until we figure out a better build system. I'm hoping to let zach and tom figure out blaze for hubris tasks then copy that.